common/src/pcr.rs

```diff
@@ -0,0 +1,64 @@
+/*++
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    pcr.rs
+
+Abstract:
+
+    PCR-related types.
+
+--*/
+
+use zerocopy::{AsBytes, FromBytes};
+
+// PcrLogEntryId is used to identify the PCR entry.
```

Comment:
Consider, "used to identify the PCR entry's semantics and size."

Reply:
Updated.